codeHF/Compare.C

```diff
@@ -42,9 +42,9 @@ Int_t Compare(TString filerun3 = "AnalysisResults_O2.root", TString filerun2 = "
   // Histogram specification: axis label, AliPhysics name, O2Physics path/name, rebin, log scale histogram, log scale ratio, projection axis
 
   VecSpecHis vecHisEvents;
-  // AddHistogram(vecHisEvents, "primary vtx x (cm)", "hPrimVertX", "hf-tag-sel-collisions/hPrimVtxX", 1, 1, 0);
-  // AddHistogram(vecHisEvents, "primary vtx y (cm)", "hPrimVertY", "hf-tag-sel-collisions/hPrimVtxY", 1, 1, 0);
-  // AddHistogram(vecHisEvents, "primary vtx z (cm)", "hPrimVertZ", "hf-tag-sel-collisions/hPrimVtxZ", 1, 1, 0);
```

Comment:
Why are the same lines commented here?

Reply:
These histograms were not included previously but I think it would not hurt to look at them in our validation studies, that is why I include them now.